models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - mandt
+            - pernr
+            - subty
+            - objps
+            - sprps
+            - endda
+            - begda
+            - seqnr
+    columns:
+      - name: mandt
+        description: Client
+      - name: pernr
+        description: Personnel number
+      - name: subty
+        description: Subtype
+      - name: objps
+        description: Object identification
+      - name: sprps
+        description: Lock indicator for HR master data record
+      - name: begda
+        description: Start date
+      - name: endda
+        description: End date
+      - name: seqnr
+        description: Number of infotype record with same key
+      - name: aedtm
+        description: Changed on
+      - name: ancur
+        description: Currency key for annual salary
+      - name: ansal
+        description: Annual salary
+      - name: anz01
+        description: Number
+      - name: anz02
+        description: Number
+      - name: anz03
+        description: Number
+      - name: anz04
+        description: Number
+      - name: anz05
+        description: Number 
+      - name: anz06
+        description: Number
+      - name: anz07
+        description: Number
+      - name: anz08
+        description: Number
+      - name: anz09
+        description: Number
+      - name: anz10
+        description: Number
+      - name: anz11
+        description: Number
+      - name: anz12
+        description: Number
+      - name: anz13
+        description: Number
+      - name: anz14
+        description: Number
+      - name: anz15
+        description: Number
+      - name: anz16
+        description: Number
+      - name: anz17
+        description: Number
+      - name: anz18
+        description: Number
+      - name: anz19
+        description: Number
+      - name: anz20
+        description: Number
+      - name: anz21
+        description: Number
+      - name: anz22
+        description: Number
+      - name: anz23
+        description: Number
+      - name: anz24
+        description: Number
+      - name: anz25
+        description: Number
+      - name: anz26
+        description: Number
+      - name: anz27
+        description: Number
+      - name: anz28
+        description: Number
+      - name: anz29
+        description: Number
+      - name: anz30
+        description: Number
+      - name: anz31
+        description: Number
+      - name: anz32
+        description: Number
+      - name: anz33
+        description: Number
+      - name: anz34
+        description: Number
+      - name: anz35
+        description: Number
+      - name: anz36
+        description: Number
+      - name: anz37
+        description: Number
+      - name: anz38
+        description: Number
+      - name: anz39
+        description: Number
+      - name: anz40
+        description: Number
+      - name: bet01
```

Comment:
Are these all intended to be the same description?

Reply:
Will update this to number/unit
<img width="695" alt="Screenshot 2023-07-06 at 4 20 19 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/6b6a616a-3d48-4ec6-892b-f3ed3a0bab7f">